src/stories/DatePicker.stories.tsx

```diff
@@ -26,6 +26,7 @@ export const DatePickerExample = Template.bind({});
 DatePickerExample.args = {
   name: "name",
   label: "DatePickerInput Label",
+  disabled: false,
```

Comment:
Are those defaults even needed?

Reply:
Unfortunately the story does not recognize the internal defaults.
If I don't add them, the storybook will display: `disabled` | `set boolean`
![image](https://user-images.githubusercontent.com/70900867/212046949-0b190cf3-f04f-495f-859b-99ab4432fa21.png)
instead of `disabled` | `false/true`
![image](https://user-images.githubusercontent.com/70900867/212047260-26b4ded7-9926-4ef5-b469-083726c72e2f.png)
Which is something I wanted to avoid, since the default value should be `false`

Reply:
Thank you for the explanation. I get your point. 
For me it would also be okay with the `set boolean` button since it's still defaulting to `false` after pressing it. 

But I have no strong opinion about it. So you can also leave it as it is right now :-) 

Thanks!

Reply:
I'll leave it as is, I like it being set already 😁